corespace-ehrekonto/ehrekonto_api
# Vscode development environment - docker setup

# Notes

The project now supports a docker dev environment setup for vscode.
This adds following to the repo:
- .devcontainer
  | - devcontainer.json
  | - Dockerfile

### Important do not under any circumstances change or edit the devcontainer.json or Dockerfile inside of the .devcontainer folder!!!! This would break the project if commited

### Sidenote: The project setup is still the same as before
- Install npm dependencies (npm install)
- Add the environment file (.env) in the project root (use the .env example use in the readme)
- Now you should be ready to go

# Setup

- pull the latest commit in the dev branch
- open the project in vscode (there should now be a hint for the remote-container)
- press the open in remote-container (wait until environment has been build)
- get coding

# Other Changes
- removed password return on user create
- removed template routes
- added honor and user route
- image handling changed to corespace cdn
